### DISCUSSION TITLE
Map/skin import and osu!direct issues

### DISCUSSION THREAD

**Discussion by JayBean26** [ppy/osu#15069] [Q&A] [answered]
When I try to import maps from stable it gets to about 100 of the 500 I have before stopping and saying it's failed, not telling me why
![image](https://user-images.githubusercontent.com/92402968/137030538-a3ce0ac5-0c07-48c8-957d-9437ff2b0ba7.png)

Skin imports straight up fail when trying to import from stable too.

osu!direct also just doesn't load, period. (circle of death in the main menu)
![image](https://user-images.githubusercontent.com/92402968/137030743-56f9ffa4-66b9-4e8b-a6d3-416d81c30cfd.png)
and just gives me an error when I try and download elsewhere (like here I tried to initiate a download through playlists) 
![image](https://user-images.githubusercontent.com/92402968/137030933-63d419c2-4f61-4f80-91d6-f0b56afb31c3.png)

**Comment by bdach** [comment, marked as answer]
Please attach logs from either:
* `%AppData%/osu/logs` (on Windows),
* `~/.local/share/osu/logs` (on Linux and macOS),
* `Android/Data/sh.ppy.osulazer/logs` (on Android).

**Reply by JayBean26** [reply]
[database.log](https://github.com/ppy/osu/files/7333458/database.log)
[network.log](https://github.com/ppy/osu/files/7333459/network.log)
[performance.log](https://github.com/ppy/osu/files/7333460/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7333461/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7333462/updater.log)

**Reply by peppy** [reply]
```2021-10-12 21:41:15 [error]: System.IO.IOException: There is not enough space on the disk. : 'C:\Users\jayle\AppData\Roaming\osu\cache\fonts\c2cac6ecbf1220314b6c0ef033ad77d4#5596de0532694d495af0b84ffd7fc891#1024#1024'```

Please make some more disk space and try again.
